core/bones/base.py

```diff
@@ -883,6 +990,26 @@ def iter_bone_value(
             else:
                 yield None, None, value
 
+    def _compute(self, skel: 'viur.core.skeleton.SkeletonInstance', name: str):
+        """Performs the evaluation of a bone configured as compute"""
+
+        if "skel" not in inspect.signature(self.compute.fn).parameters:
+            # call without any arguments
+            ret = self.compute.fn()
+        else:
+            # otherwise, call with a clone of the skeleton
+            cloned_skel = skel.clone()
+            cloned_skel[name] = None  # remove bone to avoid endless recursion
+            ret = self.compute.fn(skel=cloned_skel)
```

Comment:
I'm not happy with this, but there's no better solution IMHO, talked to @ArneGudermann previously on it.